lib/ash/error/error.ex

```diff
@@ -27,6 +44,11 @@ defmodule Ash.Error do
     to_error(value, Keyword.put(opts, :stacktrace, stacktrace))
   end
 
+  @doc """
+  Converts a value to an Ash.Error type.
+  """
+  # @spec to_error_class(ash_error_fields() | term() | [ash_error_fields()] | [term()], Keyword.t()) :: ash_errors() | [ash_errors()]
```

Comment:
Hmm...it should be returning `t()` every time. Is it not? the `t()` is provided by splode itself.